src/schema/sources.ts

```diff
@@ -904,6 +933,12 @@ interface SourcesArgs extends ConnectionArguments {
 
 interface SourcesByTag extends ConnectionArguments {
   tag: string;
+  excludeSources?: string[];
```

Comment:
Btw. what do we need `excludeSources` arg from on the frontend?

Reply:
It's more of a additional thing as I build it to function as a paginated approach.
(initially the source-related-sources) would use it, but ended up not using it.

However we might find some case where we want to re-use it and have the functionality.

Reply:
👍 ok, let's see while you build it, just looked that we might be able to get around without it in the first place if unused and also to limit what can be sent as args on these kind of paginated feed queries.